docs/api/api-reference.mdx

```diff
@@ -2,6 +2,12 @@
 title: API Reference
 ---
 
+:::note
+
+At this time, not all Rancher resources have been made available through the Rancher Kubernetes API.
```

Comment:
```suggestion
At this time, not all Rancher resources are available through the Rancher Kubernetes API.
```

Just suggesting a word choice edit to avoid passive voice